src/app/screens/Record/eventHandlers/eventHandlers.js

```diff
@@ -0,0 +1,24 @@
+import OperationActions from 'redux/operations/actions';
```

Comment:
Delete the eventHandlers folder and move this file into a utils.js file on src/app/screens/Record/